docs/install-synthesize.rst

```diff
@@ -1,6 +1,15 @@
 Installing From Synthesize
 ==========================
 
-`Synthesize <https://github.com/obfuscurity/synthesize/>`_ is a script dedicated to making Graphite easy to install. As of this writing, the default installation provides Graphite 0.9.15 for Ubuntu Linux 14.04 LTS with an experimental release candidate for tracking Graphite ``HEAD``. Users may run the installation script manually, or they can choose to use the provided Vagrantfile.
+`Synthesize <https://github.com/obfuscurity/synthesize/>`_ is a script dedicated to making Graphite easy to install. As of this writing, the default installation provides Graphite 1.1.7 for Ubuntu Linux 18.04 LTS with an experimental release candidate for tracking Graphite ``HEAD``. Users may run the installation script manually, or they can choose to use the provided Vagrantfile.
 
 For detailed instructions, please refer to the official project documentation on the `Synthesize <https://github.com/obfuscurity/synthesize/>`_ website.
+
+Installing From REsynthesize
+==========================
```

Comment:
Travis is broken:

Warning, treated as error:

/home/travis/build/graphite-project/graphite-web/docs/install-synthesize.rst:9: WARNING: Title underline too short.

Installing From REsynthesize

==========================

ERROR: InvocationError for command /home/travis/build/graphite-project/graphite-web/.tox/docs/bin/sphinx-build -W -b html -d /home/travis/build/graphite-project/graphite-web/.tox/docs/tmp/doctrees . /home/travis/build/graphite-project/graphite-web/.tox/docs/tmp/html (exited with code 1)